pyatlan/events/atlan_lambda_handler.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Atlan Pte. Ltd.
+import json
+import os
+
+from pyatlan.events.atlan_event_handler import (
+    AtlanEventHandler,
+    is_validation_request,
+    valid_signature,
+)
+from pyatlan.model.assets import Asset
+from pyatlan.model.events import AtlanEvent, AtlanEventPayload
+
+SIGNING_SECRET = str(os.environ.get("SIGNING_SECRET"))
+
+
+def process_event(handler: AtlanEventHandler, event, context):
+    body = event.get("body")
+    if is_validation_request(body):
+        print("Matches a validation request - doing nothing and succeeding.")
+        return {"statusCode": 200}
+    else:
+        if valid_signature(SIGNING_SECRET, event.get("headers")):
```

Comment:
It changed the if to if not and move up raising the exception. Then the else would not be needed:
``` if not valid_signature(SIGNING_SECRET, event.get("headers")):```
```    raise IOError("Invalid signing secret received - will not process this request.")```